compose.go

```diff
@@ -75,13 +75,20 @@ func CreateProject(name string, composeFiles ...string) (*Project, error) {
 }
 
 // Start creates and starts the compose project.
-func (p *Project) Start() error {
+func (p *Project) Start(services ...string) error {
 	ctx := context.Background()
 	err := p.composeProject.Create(ctx, options.Create{})
 	if err != nil {
 		return err
 	}
-	err = p.composeProject.Start(ctx)
+
+	return p.UnPause(services...)
+}
+
+// UnPause only starts created services which are stopped.
+func (p *Project) UnPause(services ...string) error {
```

Comment:
🤔  `pause`/`unpause` are already docker and docker-compose commands so I think it's the wrong name for the method here (i.e. it's going to be confusing).

Reply:
Also, does it need to be exported ? 👼